ISSUE_TEMPLATE.md

```diff
@@ -2,16 +2,17 @@
 *Please read the contribution guidelines first!*
```

Comment:
Let's add a note here about known issues and maybe link to them. I know this is redundant if you read the contribution guidelines, but people may not do that, so the lower we can make the barrier to entry the better. Maybe link to the contribution guidelines again as well.